src/pytest_dbfixtures/factories/mysql.py

```diff
@@ -104,7 +105,7 @@ def mysql_proc_fixture(request):
         mysql_host = host or config.mysql.host
         mysql_params = params or config.mysql.params
 
-        tmpdir = path(gettempdir())
+        tmpdir = path(mkdtemp(prefix="pytest-mysql-"))
```

Comment:
why the gettempdir wouldn't be appropriate? 
can't we pass it to `--tmpdir` to mysql commands?

Note, that the gettempdir is affected by TMPDIR environment variable. see https://docs.python.org/2/library/tempfile.html#tempfile.gettempdir